.github/workflows/build.yml

```diff
@@ -9,18 +9,18 @@ on:
 permissions:
   contents: read
 
-env:
-  HATCHET_APP_LIMIT: 100
-  HATCHET_RETRIES: 3
-  HEROKU_API_KEY: ${{ env.HEROKU_API_KEY }}
-  HEROKU_API_USER: ${{ env.HEROKU_API_USER }}
-  IS_RUNNING_ON_CI: 1
-  HATCHET_EXPENSIVE_MODE: 1
 jobs:
-  ruby-2.2:
+  ruby-2-7:
```

Comment:
GitHub Action's `matrix` can be used to save the duplication between Ruby versions.

I would also recommend using https://github.com/ruby/setup-ruby rather than running inside the container (which is what the conversion tool used). 

That action installs dependencies for you, and also has built-in caching, so the extra caching steps could then be removed as well.

For example:

```yaml
jobs:
  test:
    strategy:
      fail-fast: false
      matrix:
        ruby-version: ["2.7", "3.0", "3.1"]
    runs-on: ubuntu-22.04
    env:
      # ...
    steps:
      - name: Checkout
        uses: actions/checkout@v3
      - name: Set up Ruby ${{ matrix.ruby-version }} and dependencies
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: ${{ matrix.ruby-version }}
          bundler-cache: true
      - name: Hatchet setup
        run: bundle exec hatchet ci:setup
      - name: Run test suite
        run: bundle exec parallel_split_test spec/
```

Reply:
Done